go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/magiconair/properties v1.8.7
 	github.com/moby/term v0.5.0
 	github.com/opencontainers/image-spec v1.1.0-rc4
+	github.com/spf13/cobra v1.7.0
```

Comment:
As a consequence of my previous comment, cobra would not be added as dependency to the core, but instead to the `cmd` package/module